### PR TITLE
Allow users to specify the attribute name

### LINKED ISSUE #8: Including dates?
I was considering starting a branch/pr to allow the user to pass a key for the name of the column so that you could also use this for dates:

```ruby
time_for_a_boolean :stale, :stale_on
```

Is there a reason you wouldn't want this ability? I'd like to use this when I'm storing pure date columns, not date/time but I'm curious if there is a reason I shouldn't do that

### PR DESCRIPTION
**Pull request by drapergeek**
This allows user to use a column in the database that is named
something other than `attribute_at`. On the surface,
the primary usage would be if a user wanted to use a date column,
they would follow the convention of using `attribute_on`,
then specify that name in the `time_for_a_boolean column.
It also allows you to simply name your column whatever you prefer.

We probably shouldn't force people to follow conventions,
their column names may be out of their control.

Fixes #8

### CODE REVIEW

**Comment by drapergeek** [comment]
*cough* @calebthompson *cough*

File: README.md [hunk 1]
```diff
@@ -77,3 +77,16 @@ Okay... why?
 * `COUNT(posts.deleted_at)` gives you the count of deleted posts, which is
   useful when writing a report. Define and use `Post.deleted.count` when you
   have Ruby available.
+
+Other Options
+-------------
+
+If you have a date or time column
+that does not follow the `attribute_at` convention,
+you can specify the attribute name:
```
Comment (calebthompson) [inline README.md]: Can you:

1. Match the convention of wrapping at 80 chars
2. Mention that this is useful for date fields

File: lib/time_for_a_boolean.rb [hunk 2]
```diff
@@ -4,17 +4,22 @@
 require "time_for_a_boolean/railtie"
 
 module TimeForABoolean
-  def time_for_a_boolean(attribute)
+  def time_for_a_boolean(attribute, backing_attribute_name=nil)
+    backing_attribute_name ||= "#{attribute}_at"
```
Comment (calebthompson) [inline lib/time_for_a_boolean.rb]: What do you think of `time_for_a_boolean(name, field="#{name}_at") for the signature and variable names?

```
2.2.3 :001 > def tfab(name, field="#{name}_at")
2.2.3 :002?>   puts name, field
2.2.3 :003?>   end
 => :tfab 
2.2.3 :004 > tfab "approved"
approved
approved_at
 => nil 
2.2.3 :005 > tfab "published", "published_on"
published
published_on
```

Comment (drapergeek) [inline lib/time_for_a_boolean.rb]: I did not know you could reference the first argument in the second argument!

Comment (calebthompson) [inline lib/time_for_a_boolean.rb]: It's cool, right? I was pretty sure I'd tried it before but had to confirm.

**Comment by drapergeek** [comment]
Updated per requests!

File: lib/time_for_a_boolean.rb [hunk 3]
```diff
@@ -4,17 +4,19 @@
 require "time_for_a_boolean/railtie"
 
 module TimeForABoolean
-  def time_for_a_boolean(attribute)
+  def time_for_a_boolean(attribute, field="#{attribute}_at")
     define_method(attribute) do
-      !send("#{attribute}_at").nil? &&
-        send("#{attribute}_at") <= -> { Time.current }.()
+      !send(field).nil? && send(field) <= -> { Time.current }.()
```
Comment (calebthompson) [inline lib/time_for_a_boolean.rb]: nice.

**Comment by calebthompson** [comment]
Cool, I'll wait for the tests and merge.

**Comment by calebthompson** [comment]
Looks like it failed. Can you have a look (and squash when it's fixed)?

**Comment by drapergeek** [comment]
>Looks like it failed. Can you have a look (and squash when it's fixed)?

Doesn't look to be a legit failure and it passes locally.

Just pushed again, we'll see if that takes care of it.

**Comment by drapergeek** [comment]
Yo @calebthompson mind taking a look at this?

**Comment by calebthompson** [comment]
I'm seeing this error on two different gems on Travis. I'm blaming travis now.

**Pull request merged**